packages/angular_devkit/core/package.json

```diff
@@ -11,7 +11,9 @@
     "core"
   ],
   "dependencies": {
-    "ajv": "~5.5.1",
+    "ajv": "^6.1.0",
+    "ajv-async": "^1.0.1",
```

Comment:
Question; if we limit to Node 8+ do we still need ajv-async?

Reply:
We do not, no. But we'll need update appveyor to use node 8 as well.